drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -6905,7 +6905,11 @@ void RasterizerStorageGLES3::initialize() {
 	config.s3tc_supported = config.extensions.has("GL_EXT_texture_compression_dxt1") || config.extensions.has("GL_EXT_texture_compression_s3tc") || config.extensions.has("WEBGL_compressed_texture_s3tc");
 	config.etc_supported = config.extensions.has("GL_OES_compressed_ETC1_RGB8_texture");
 	config.latc_supported = config.extensions.has("GL_EXT_texture_compression_latc");
-	config.rgtc_supported = config.extensions.has("GL_EXT_texture_compression_rgtc");
+#ifdef OSX_ENABLED
+	config.rgtc_supported = true;
```

Comment:
It is supported by all current mac GPUs [OpenGL-Capabilities-Tables](https://developer.apple.com/opengl/OpenGL-Capabilities-Tables.pdf) and can't be detected by extension name, Apple's drivers only list extensions that aren't core in the current context version.